apprentice-react-o-shop/.gitignore

```diff
@@ -0,0 +1,2 @@
+# dependencies
+/node_modules
```

Comment:
remove comments in this file and append a new line at the end of the file. Otherwise github gives you that weird no new line symbol